test/cases/api/common/s3.sh

```diff
@@ -383,6 +383,12 @@ function verifyInVSphere() {
         -on=false \
         "${VSPHERE_VM_NAME}"
 
+    # tagging vm as testing object
+    $GOVC_CMD tags.attach -u "${GOVMOMI_USERNAME}":"${GOVMOMI_PASSWORD}"@"${GOVMOMI_URL}" \
```

Comment:
extreme nitpick: you could have ended the first line after `$GOVC_CMD tags.attach` and have the `-u ...` part on the new line, to make it consistent with all the other commands above and below. I just wanted to point this out 😅